core/domain/fs_domain.py

```diff
@@ -107,26 +116,152 @@ def _validate_entity_parameters(self, entity_name, entity_id):
             raise utils.ValidationError('Entity id cannot be empty')
 
     @property
-    def assets_path(self):
+    def assets_path(self) -> str:
         """Returns the path of the parent folder of assets.
 
         Returns:
             str. The path.
         """
         return self._assets_path
 
+    def _get_gcs_file_url(self, filepath: str) -> str:
+        """Returns the constructed GCS file URL.
+
+        Args:
+            filepath: str. The path to the relevant file within the entity's
+                assets folder.
+
+        Returns:
+            str. The GCS file URL.
+        """
+        # Upload to GCS bucket with filepath
+        # "<entity>/<entity-id>/assets/<filepath>".
+        gcs_file_url = '%s/%s' % (self._assets_path, filepath)
+        return gcs_file_url
+
+    def isfile(self, filepath: str) -> bool:
+        """Checks if the file with the given filepath exists in the GCS.
+
+        Args:
+            filepath: str. The path to the relevant file within the entity's
+                assets folder.
+
+        Returns:
+            bool. Whether the file exists in GCS.
+        """
+        return storage_services.isfile(
+            self._bucket_name, self._get_gcs_file_url(filepath))
```

Comment:
This should not be implemented here. Just add `NotImplementedError` to these functions in this class.